test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -37,117 +86,81 @@ define([
-                    optional: 1,
-                    short_hint: 'Short read file containing a paired end library in FASTA/FASTQ format',
-                    ui_class: 'parameter',
-                    ui_name: 'Forward/Left FASTA/FASTQ File Path'
-                },
-                ui: {
-                    advanced: false,
-                    class: 'parameter',
-                    control: 'dynamic_dropdown',
-                    description: 'Valid file extensions for FASTA: .fasta, .fna, .fa   Valid file extensions for FASTQ: .fastq, .fnq, .fq; Compressed files (containing files with vaild extentions): .zip, .gz, .bz2, .tar.gz, .tar.bz2',
-                    hint: 'Short read file containing a paired end library in FASTA/FASTQ format',
-                    label: 'Forward/Left FASTA/FASTQ File Path',
-                    type: 'dynamic_dropdown'
-                }
-            };
-
-            paramResolver = ParamResolver.make();
-
-            inputControlFacotry = paramResolver.loadInputControl(parameterSpec);
-
             const model = Props.make({
                 data: TestAppObject,
-                onUpdate: (props) => {},
+                onUpdate: () => {},
             });
 
-            appSpec = Spec.make({
+            const appSpec = Spec.make({
                 appSpec: model.getItem('app.spec'),
             });
 
-            closeParameters = [
-                'fastq_fwd_staging_file_name',
-                'fastq_rev_staging_file_name',
-                'sra_staging_file_name',
-                'name'
-            ];
-
-            return mockFieldWidget = fieldCellWidget.make({
-                inputControlFactory: inputControlFacotry,
-                showHint: true,
-                useRowHighight: true,
-                initialValue: '',
-                appSpec: appSpec,
-                parameterSpec: parameterSpec,
-                workspaceId: 56236,
-                referenceType: 'name',
-                paramsChannelName: 'Test Channel',
-                closeParameters: closeParameters
+            const paramResolver = ParamResolver.make();
+
+            await paramResolver.loadInputControl(parameterSpec).then((inputControlFactory) => {
+                return (mockFieldWidget = fieldCellWidget.make({
+                    inputControlFactory: inputControlFactory,
+                    showHint: true,
+                    useRowHighight: true,
+                    initialValue: '',
+                    appSpec: appSpec,
+                    parameterSpec: parameterSpec,
+                    workspaceId: 56236,
+                    referenceType: 'name',
+                    paramsChannelName: 'Test Channel',
+                    closeParameters: closeParameters,
+                }));
             });
         });
 
-        it('has a factory which can be invoked', function() {
+        afterEach(async () => {
+            await mockFieldWidget.stop().catch((err) => {
```

Comment:
trying to clear things out after each run, not sure if this is the best way to do it?